card/db/schema.rb

```diff
@@ -38,7 +38,7 @@
   create_table "card_changes", force: :cascade do |t|
     t.integer "card_action_id", limit: 4
     t.integer "field",          limit: 4
-    t.text    "value",          limit: 65535
+    t.text    "value",          limit: 16777215
```

Comment:
is it safe to assume that we started doing this because rails defaults shifted, not because somebody decided it was a good idea?